scripts/run-in-pipeline.sh

```diff
@@ -0,0 +1,197 @@
+#     - default value for that same parameter
+#   Output:
+#     - case-insentive value (true/false) or default value
+##
+function normalize {
+    local user_param=$1
+    local default_val=$2
+
+    if [[ "${user_param,,}" == "$_true" ]]  # Note:  ${var,,} means to lowercase of var
+    then
+        result="$_true"
+    elif [[ "${user_param,,}" == "$_false" ]]
+    then
+        result="$_false"
+    else
+        result="$default_val"
+    fi
+
+    echo $result
+}
+
+## Initialize Detect Secrets audit_report parameter string
+audit_report_params=' '
+
+##
+# Starting the pipeline Detect Secrets run
+##
+echo "[ Starting Detect Secrets run ]"
+echo
+echo "...using Baseline: $BASELINE"
+
+##
+# Checking Env Vars and appending coresponding Detect Secrets audit report parameters to the parameter string
+# Note:  Env Var input values are normalized before being checked. Thus case-insensitive and/or defaulted if necessary
+#        Thus the values will always be either true or false after they are normalized
+##
+
+# skip_scan parameter
+skip_scan="$(normalize $skip_scan $_skip_scan_default)"
+if [[ "$skip_scan" == "$_true" ]]
+then
+  echo "...skip scan with baseline update: $_true"
+else
+  echo "...skip scan with baseline update: $_false"
+fi
+
+# json parameter
+json="$(normalize $json $_json_default)"
+if [[ "$json" == "$_true" ]]
+then
+  echo "...output json: $_true"
+  audit_report_params="$audit_report_params $_json_option"
+else
+  echo "...output json: $_false"
+fi
+
+# omit_instructions parameter
+omit_instructions="$(normalize $omit_instructions $_omit_instructions_default)"
+if [[ "$omit_instructions" == "$_true" ]]
+then
+  echo "...omit instructions: $_true"
+  audit_report_params="$audit_report_params $_omit_instructions_option"
+else
+  echo "...omit instructions: $_false"
+fi
+
+
+# Fail On Live
+fail_live="$(normalize $fail_live $_fail_live_default)"
+if [[ "$fail_live" == "$_true" ]]
+then
+  echo "...fail on live: $_true"
+  audit_report_params="$audit_report_params $_fail_live_option"
+else
+  echo "...fail on live: $_false"
+fi
+
+# Fail On Unaudited
+fail_unaudited="$(normalize $fail_unaudited $_fail_unaudited_default)"
+if [[ "$fail_unaudited" == "$_true" ]]
+then
+  echo "...fail on unaudited: $_true"
+  audit_report_params="$audit_report_params $_fail_unaudited_option"
+else
+  echo "...fail on unaudited: $_false"
+fi
+# Fail On Audited Real
+fail_audited_real="$(normalize $fail_audited_real $_fail_audited_real_default)"
+if [[ "$fail_audited_real" == "$_true" ]]
+then
+  echo "...fail on audited real: $_true"
+  audit_report_params="$audit_report_params $_fail_audited_real_option"
+else
+  echo "...fail on audited real: $_false"
+fi
+
+## Calling Detect Secrets scan with baseline update to create or update existing baseline file
+if [[ "$skip_scan" == "$_false" ]]
+then
+  echo "Scanning code directory (docker volume mounted to $PWD) with baseline update ... "
```

Comment:
being more specific here:
```suggestion
  echo "Scanning code directory (docker volume mounted to $PWD) and updating baseline file $baseline ... "
```